### ISSUE TITLE
PID 1 zombie problem

### ISSUE THREAD

**Issue by truman-misfit** [Misfit-Wearables/sw_cloud_dockers#1]
As described in the blog [Docker and the PID 1 zombie reaping problem](https://blog.phusion.nl/2015/01/20/docker-and-the-pid-1-zombie-reaping-problem/), I think we should change our base image from ubuntu:14.04 to the phusion/baseimage:latest. Since the phusion/baseimage adds some lines looks like the part below:
```
#!/bin/bash
function cleanup()
{
    local pids=`jobs -p`
    if [[ "$pids" != "" ]]; then
        kill $pids >/dev/null 2>/dev/null
    fi
}

trap cleanup EXIT
/path-to-your-app
```

The details about PID 1 zombie problem has been described in the [phusion.github.io](https://phusion.github.io/baseimage-docker/).

**Comment by forhot2000** [comment]
this bug has fix since docker v1.5, so don't worry about it.

**Comment by truman-misfit** [comment]
Got it and thanks
